Cipher.js

```diff
@@ -188,6 +188,59 @@ export class Cipher {
     return JSON.parse(new TextDecoder().decode(data));
   }
 
+  /**
+   * Creates a JWE recipient using the given inputs.
+   *
+   * @see https://tools.ietf.org/html/rfc7516#section-4
+   *
+   * @param {object} options - Options to use.
+   * @param {object} options.recipient - A recipient with a header with a
+   *   kid and alg.
+   * @param {object} options.ephemeralKeyPair - An ephemeral key pair.
+   * @param {object} options.cek - A content encryption key.
+   * @param {Function} options.keyResolver - A function that can resolve keys.
+   *
+   * @returns {Promise<object>} A JWE recipient object.
+   */
+  async createRecipient({recipient, ephemeralKeyPair, cek, keyResolver}) {
```

Comment:
Let's make this a private function and move it into the private function area (bottom of file) for now since we haven't worked out how a `cek` would be passed in or anything. Then I think this PR can be merged.

Reply:
@dlongley done.